docs/references/astro/organization-store.mdx

```diff
@@ -0,0 +1,110 @@
+
+```tsx filename="session.tsx"
+import { useStore } from '@nanostores/react';
+import { $organizationStore } from '@clerk/astro/client';
+
+export default function Home() {
+  const organization = useStore($organizationStore);
+
+  if (organization === undefined) {
+    // Add logic to handle loading state
+    return null;
+  }
+
+  if(organization === null) {
+    // Add logic to handle no active organization state
+    return null;
+  }
+
+  return (
+    <div>
+      <p>This current organization is {organization.name}</p>
+    </div>
+  )
+}
+```
+
+## Paginating data
+
+The following example demonstrates how to implement pagination for organization memberships. The `memberships` state will be populated with the first page of the organization's memberships. When the "Previous page" or "Next page" button is clicked, the `fetchMemberships` function will be called to fetch the previous or next page of memberships.
+
+You can implement this pattern to any Clerk function that returns a [`ClerkPaginatedResponse`](/docs/references/javascript/types/clerk-paginated-response#clerk-paginated-response) object.
+
+```tsx filename="organization-members.tsx"
+import { useState, useEffect } from 'react';
+import { $organizationStore } from '@clerk/astro/client'
+import { useStore } from '@nanostores/react';
+
+export default function OrganizationMembers() {
+  const [memberships, setMemberships] = useState([]);
+  const [currentPage, setCurrentPage] = useState(1);
+  const organization = useStore($organizationStore);
+
+  const pageSize = 10;
+
+  useEffect(() => {
+    fetchMemberships();
+  }, [currentPage, organization]);
+
+  const fetchMemberships = async () => {
+    if (!organization) {
+      return;
+    }
+
+    try {
+      const { data } = await organization.getMemberships({
+        initialPage: currentPage,
+        pageSize: pageSize
+      });
+      setMemberships(data);
+    } catch (error) {
+      console.error("Error fetching memberships:", error);
+    }
+  };
+
+  const fetchPrevious = () => {
+    if (currentPage > 1) {
+      setCurrentPage(currentPage - 1);
+    }
+  };
+
+  const fetchNext = () => {
+    setCurrentPage(currentPage + 1);
+  };
+
+  if (!organization) {
+    // Handle loading state
+    return null;
+  }
+
+  return (
+    <div>
+      <h2>Organization members</h2>
+      <ul>
+        {memberships.map((membership) => (
+          <li key={membership.id}>
+            {membership.publicUserData.firstName} {membership.publicUserData.lastName} &lt;
+            {membership.publicUserData.identifier}&gt; :: {membership.role}
+          </li>
+        ))}
+      </ul>
+      <div>
+        <button onClick={fetchPrevious} disabled={currentPage === 1}>Previous page</button>
+        <button onClick={fetchNext}>Next page</button>
+      </div>
+    </div>
+  );
+};
+```
```

Comment:
@royanger Updated and added an example on how to paginate data and a note that they can implement this pattern on other functions that returns a [`PaginatedResponse`](https://clerk.com/docs/references/javascript/types/clerk-paginated-response#clerk-paginated-response). They can use this to mimic `useOrganizationList`. Would this suffice?

Reply:
This looks pretty solid.

Reply:
Thanks for reviewing!